### PR TITLE
Add zensummoning documentation

### PR DESCRIPTION
**Pull request by TeamDman**
This is mostly autogenerated, and I'm not sure what else needs to be added for sidebar and i18n support.

### CODE REVIEW

File: docs/mods/zensummoning/MobInfo.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+Creates a new MobInfo with default values. <br />  See other methods for adding more customization.
+
+Returns: new MobInfo  
+Return Type: [MobInfo](/mods/zensummoning/MobInfo)
+
+```zenscript
+// MobInfo.create() as MobInfo
+
+MobInfo.create();
```
Comment (kindlich) [inline docs/mods/zensummoning/MobInfo.md:25]: You could also just annotate the constructor with `@ZenCodeType.Constructor` 😉

File: docs/mods/zensummoning/MobInfo.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+```zenscript
+// MobInfo.setMob(mob as string) as MobInfo
+
+myMobInfo.setMob("minecraft:zombie_villager");
+```
+
+| Parameter | Type | Description |
+|-----------|------|-------------|
+| mob | string | resource location |
```
Comment (kindlich) [inline docs/mods/zensummoning/MobInfo.md:94]: You can use MCEntityType here, for a bit more of type safety:
`myMobInfo.setMob(<entityType:minecraft:zombie_villager>)`

**Review by kindlich — CHANGES_REQUESTED** [review]
Changes to docs.json missing.
You may want to move your stuff to `docs_exported` if they are only generated by the AP and have your example an extra file here 🤔 

@jaredlll08 what do you think?

**Comment by jaredlll08** [comment]
As kindlich said, you can just move the generated files into like ```docs_exported/ZenSummoning``` and it should handle the rest of merging the docs.json file

**Comment by jaredlll08** [comment]
For the examples, can you just add a little "The following script will do: <explain what it does", just so people know what they are looking at.

Otherwise it looks great!

File: docs/mods/zensummoning/examples.md [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+
+Players can look at the usage of the altar in JEI to see available summonings.  
+Looking at the recipes for spawn eggs will show summonings that include the mob the egg is for.
+
+## Example summoning 1
+
+- Summons 4 cows and 10 blazes
+    - Spawns the blazes 8 blocks above the altar, +/- 3 blocks in each direction
+    - Spawns the cows 4 blocks above the altar, +/- 3 blocks in each direction
+- Consumes 1 stone, 12 eggs, 1 stick as catalyst
```
Comment (jaredlll08) [inline docs/mods/zensummoning/examples.md:16]:
```suggestion
- Consumes 1 stone, 12 eggs and 1 stick as catalyst
```

File: docs/mods/zensummoning/examples.md [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+- Doesn't work in the rain
+    - If it's raining, displays a custom message
+    - If it's successful, displays "Woohoo!"
+- Gives the mobs custom NBT
+    - 1.16 renamed generic.maxHealth to generic.max_health.
+    - https://minecraft.fandom.com/wiki/Attribute
```
Comment (jaredlll08) [inline docs/mods/zensummoning/examples.md:22]:
```suggestion
    - [https://minecraft.fandom.com/wiki/Attribute](https://minecraft.fandom.com/wiki/Attribute)
```

File: docs/mods/zensummoning/examples.md [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+                attempt.message = "Woohoo!";
+            }
+        })
+);
+```
+
+## Example summoning 2
+
+- Spawns one cow
+- Consumes one sapling (with oredict support), consumes one stone as catalyst
```
Comment (jaredlll08) [inline docs/mods/zensummoning/examples.md:64]:
```suggestion
- Consumes one sapling (with oredict support) and consumes one stone as catalyst
```

File: docs/mods/zensummoning/examples.md [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+                "PersistenceRequired":1,
+                "CustomNameVisible":1
+            })
+        )
+);
+```
+
+## Example summoning 4 and 5
+
+- Adds two summoning with same reagents an catalyst
```
Comment (jaredlll08) [inline docs/mods/zensummoning/examples.md:129]:
```suggestion
- Adds two summoning with the same reagents an catalyst
```

File: docs/mods/zensummoning/examples.md [hunk 7]
```diff
@@ -0,0 +1,233 @@
...
+    - One spawns a blaze, the other spawns a cow. Both don't work in the rain.
+- When attempted, there is a 4:1 chance that you will summon the cow.
```
Comment (jaredlll08) [inline docs/mods/zensummoning/examples.md:131]:
```suggestion
- When attempted, there is a 4 in 1 chance that you will summon the cow.
```

File: docs/mods/zensummoning/examples.md [hunk 8]
```diff
@@ -0,0 +1,233 @@
...
+            attempt.success = false;
+            attempt.message = "test!";
+        }
+    })
+);
+```
+
+## Example summoning 6
+
+- Consumes 1 stone, 12 eggs
```
Comment (jaredlll08) [inline docs/mods/zensummoning/examples.md:176]:
```suggestion
- Consumes 1 stone and 12 eggs
```

**Review by jaredlll08 — CHANGES_REQUESTED** [review]

File: docs/mods/zensummoning/examples.md (hunk 7 above)
Comment (TeamDman) [reply docs/mods/zensummoning/examples.md]: Changed to `4 in 5`

**Pull request merged**